package.json

```diff
@@ -18,6 +18,7 @@
     "del": "^2.2.2",
     "dotenv-safe": "^4.0.4",
     "electron-docs": "^3.0.2",
+    "get-crowdin-file-ids": "github:electron/get-crowdin-file-ids#get-method",
```

Comment:
Related to: https://github.com/electron/get-crowdin-file-ids/pull/6

Reply:
You fix landed in get-crowdin-file-ids@1.1.1. You should now be able to `npm i -D get-crowdin-file-ids@latest` 👍